### ISSUE TITLE
Replace PubsubMessage with POJO in ingestion-beam

### ISSUE THREAD

**Issue by relud** [mozilla/gcp-ingestion#65]

**Comment by jklukas** [comment]
Based on my understanding of coders and schemas in Beam at this point, we could likely go ahead and use POJOs as long as we annotate them with `@DefaultSchema(JavaBeanSchema.class)` or related `JavaFieldSchema`.

I have a thread going in the beam dev list about what it would take to add similar support for AutoValue classes, which would make for significantly less verbose object definitions.

**Comment by jklukas** [comment]
Note that default coders for AutoValue classes by way of schemas are merged and will be available in Beam 2.10 (currently on RC3, so released soon). We could have a type like:

```java
@DefaultSchema(AutoValueSchema.class)
@AutoValue
public abstract class TelemetryMessage {
  public abstract byte[] payload();
  @Nullable public abstract String documentType();
  ...
}
```

and Beam would be able to infer a coder for it.

**Comment by relud** [comment]
I don't think this is worth the effort a this time
